crates/cargo-contract/src/cmd/info.rs

```diff
@@ -94,15 +112,34 @@ impl InfoCommand {
 
         Ok(contract_info_of)
     }
+}
+
+#[derive(serde::Serialize)]
+struct InfoToJson {
+    trie_id: String,
+    code_hash: sp_core::H256,
+    storage_items: u32,
+    storage_item_deposit: u128,
+}
 
-    fn print_and_format_contract_info(info: ContractInfo) {
-        let convert_trie_id = hex::encode(info.trie_id.0);
-        name_value_println!("TrieId:", format!("{}", convert_trie_id));
-        name_value_println!("Code hash:", format!("{:?}", info.code_hash));
-        name_value_println!("Storage items:", format!("{:?}", info.storage_items));
+impl InfoToJson {
+    pub fn to_json(&self) -> Result<String> {
```

Comment:
For `pub` functions there should be comments.

Reply:
IMO this method does not need to exist, can just do `serde_json::to_string_pretty(info_to_json)?` inline.

Reply:
Comments Added - as discussed with @ascjones after it is ok like this atm